nengo_gui/static/components/netgraph.js

```diff
@@ -16,6 +16,8 @@ Nengo.NetGraph = function(parent, args) {
     this.offsetX = 0;          // global x,y pan offset
     this.offsetY = 0;
 
+    this.overide_positions = {};
```

Comment:
A comment explaining what this is used for would be appreciated. From what I can tell it's used for positioning the side-menu, but I'm not clear on why that required a new property to be created.